_terms/legal-eula.md

```diff
@@ -44,38 +44,104 @@ last_update: 01/10/2016
+3.3	The Service is licensed on either: (a) an indefinite fully paid up basis; or (b) on a subscription/rental basis for a limited time period for which Customer has made full payment. The terms of the license and any license access code (see below in Clause 2.4) are specified in the Order Confirmation or the Service Offer. Any upgrades and/or major version changes to Software requires the Customer to purchase a support and/or maintenance service from SixSq.
 
-4\. Limitation of Liability
+3.4	Customer will be provided with an access code for use of the Service for the duration of this EULA, subject to payment being received by SixSq for the Products continuing to be up to date, and SixSq continuing to be satisfied that the Service is being used by Customer and its End-Users in accordance with the Terms.
+
+3.5	Subject to full payment by Customer for the Service and Customer’s compliance with the Terms and this EULA, SixSq hereby grants the End-User a non-exclusive and non-transferable license to the Service on the following terms: (a) the End-User shall only use the Service for normal use as described in the Specification; (b) the End-User shall not copy (except to the extent permissible under applicable law or for normal operation of the Service), reproduce, translate, adapt, vary or modify the Service, nor communicate it to any third party, without SixSq’s prior written consent; (c) the End-User shall not rent, lease, sub-license, loan, translate, merge, adapt, vary or modify the Service; (d) the End-User shall not disassemble, de-compile, reverse engineer or create derivative works based on the whole or any part of the Service nor attempt to do any such things except to the extent permitted by applicable law; (e) the End-User shall not use the Service with any equipment other than that approved or supplied by SixSq, and shall not remove, adapt or otherwise tamper with any copyright notice, legend or logo which appears in or on the Service; and (f) the End-User shall replace the current version of the Edge Software the Service requires with any updated or upgraded version or new release provided by SixSq to Customer promptly on receipt of such version or release.
+
+3.6	End User acknowledges and agrees that: (a) the Software may not be free of bugs or errors and the existence of any minor bugs or errors shall not constitute a breach of this EULA; and (b) that any bugs, errors and/or restrictions in the use of the Software may be resolved by updating to the then current version of the Software and if the Customer chooses not to update or upgrade when recommended by SixSq then Customer shall be solely responsible and liable for bugs, errors and/or restrictions in the use of the Software; and (c) that the Software has not been developed to meet Customers individual requirements and therefore it is Customer’s responsibility to ensure that the functions of the Software as described in the Specification meet Customer’s requirements.
+
```

Comment:
shouldn't 3.6 and 3.7 be under clause 2?

Reply:
Corrected these sections to refer to Service instead of Software